hr_employee_benefit/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+{
+    'name': 'Employee Benefit',
+    'version': '11.0.1.0.0',
+    'license': 'AGPL-3',
+    'category': 'Generic Modules/Human Resources',
+    'author': "Savoir-faire Linux, Odoo Community Association (OCA)",
+    'website': 'https://www.savoirfairelinux.com',
```

Comment:
Replace `website` URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)